<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThresholdWindow</class>
 <widget class="QWidget" name="ThresholdWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ThresholdWindow</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>349</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="button_layout">
    <property name="spacing">
     <number>10</number>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QPushButton" name="addSlider">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="deleteSlider">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="undo">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="sentData">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>301</width>
     <height>351</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="top_layout">
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="spacing">
     <number>20</number>
    </property>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>addSlider</sender>
   <signal>clicked()</signal>
   <receiver>ThresholdWindow</receiver>
   <slot>add()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>56</x>
     <y>377</y>
    </hint>
    <hint type="destinationlabel">
     <x>55</x>
     <y>415</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteSlider</sender>
   <signal>clicked()</signal>
   <receiver>ThresholdWindow</receiver>
   <slot>eliminate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>183</x>
     <y>380</y>
    </hint>
    <hint type="destinationlabel">
     <x>261</x>
     <y>444</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>sentThresholds()</signal>
  <slot>add()</slot>
  <slot>eliminate()</slot>
 </slots>
</ui>
